subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -922,10 +929,12 @@ void ticker_worker(void *param)
 				/* Reset latency to periodic offset */
 				ticker->lazy_current = 0U;
 				ticker->force = 0U;
+#if !defined(CONFIG_BT_TICKER_SLOT_AGNOSTIC)
```

Comment:
Please conditionally compile out `ticks_slot` member in the `struct ticker_node` and correspondingly correct the `TICKER_NODE_T_SIZE`